va/va.h

```diff
@@ -588,6 +588,19 @@ typedef enum
      */
      VAConfigAttribEncDirtyROI       = 28,
 
+    /**
+     * \brief Parallel Rate Control (hierachical B) attribute. Read-only.
+     * This attribute conveys whether the encoder supports parallel rate control.
+     * It is a boolean value 0 - unsupported, 1 - supported.
+     * This is the way when hireachical B frames are encoded, multiple independent B frames
+     * on the same layer may be processed at same time.
+     * If supported, app may enable it by setting enable_parallel_brc in
+     * VAEncMiscParameterRateControl,
+     * and the number of B frames per layer per GOP will be passed to driver
+     * through VAEncMiscParameterParallelRateControl structure.
+     * Currently three layers are defined.
```

Comment:
How about if more than 3 layer are supported? Could you provide a mechanism to query the supported layer number.